agent/config/types.go

```diff
@@ -134,6 +134,9 @@ type Config struct {
 	// NumImagesToDeletePerCycle specifies the num of image to delete every time
 	// when Agent performs cleanup
 	NumImagesToDeletePerCycle int
+
+    // Set if clients validate ssl certificates. Used mainly for testing
+	AcceptInsecureCert bool
```

Comment:
Can you fix the formatting here?

Disabling certificate checking is a bad idea, and we shouldn't make it easier.  I'd prefer that this is not settable through the config JSON file, so can you add the `json:"-"` struct tag so this field gets ignored?

```go
AcceptInsecureCert bool `json:"-"`
```